### DISCUSSION TITLE
Anyone need Galaxy Brain?

### DISCUSSION THREAD

**Discussion by ItIsHM** [Schweinepriester/github-profile-achievements#49] [General]
Galaxy Brain

**Comment by HMser** [comment]
I need

**Comment by LokmaneKrizou** [comment]
GALAXY BRAIN 🧠

**Comment by smir45** [comment]
Yeah i need. Let's do it🧠

**Comment by ItzJustNick** [comment]
I would also like the galaxy brain achivement. I would like to thank you in advance for this thread and you contribution to the community. 😊

**Comment by Owen-Choi** [comment]
is it enough just answering? or we need 2 "selected" answer?

**Reply by HMser** [reply]
Need selected answer

**Comment by YouthguyZ** [comment]
haha i get it . you only answer 2 discussions and Marked as answer.

**Comment by devntv** [comment]
#[ItIsHM](https://github.com/ItIsHM)

**Comment by hursittarcan** [comment]
Would be nice :)

**Comment by artimys** [comment]
need of Galaxy brain

[a comment by an automated account is not shown]

**Comment by nlavanderos** [comment]
Galaaaaaxy!

**Comment by NexON39** [comment]
Yea ;)

**Comment by darakimberlys** [comment]
i want it

**Comment by HighlightCode** [comment]
nice ~ :)

**Comment by mendo94** [comment]
I am very interested in earning Galaxy brain!!

**Comment by vvscode** [comment]
I would like to have it as well =)
isn't it cheating?

**Comment by vishwajeet-hash** [comment]
How can I get it?

**Comment by adiii21** [comment]
Yes I need galaxy brain!

**Comment by adiii21** [comment]
How can I get that?

**Comment by 0xLauge** [comment]
It would be very much appreciated :)

**Comment by Patrick0105** [comment]
i need

**Comment by sourabmaity** [comment]
I need pls help me 🧠

**Comment by Thulana-Thidaswin** [comment]
Gimme

**Comment by Thulana-Thidaswin** [comment]
Do i have to post twice or?

**Comment by EeeasyCode** [comment]
yeah I need a Galaxy brains 
but i dont know how to get bedge
help me plz

**Comment by Chameleon-Coder** [comment]
I need it!
